### PR TITLE
Miscellaneous Typos

### PR DESCRIPTION
**Pull request by walkerselby**
Fixed Random Typos
Standardized spelling of SkyBlock to "SkyBlock", as indicated by Hypixel
Standardized capitalization of NEU to NEU from neu and Neu in many places
Edited messages for clarity
Expanded OF to Optifine for clarity
Other miscellaneous typos -- see commits.

### CODE REVIEW

**Comment by romangraef** [comment]
>  Standardized spelling of SkyBlock to "SkyBlock", as indicated by Hypixel
We should do the same for HyPixel

**Comment by jani270** [comment]
> > Standardized spelling of SkyBlock to "SkyBlock", as indicated by Hypixel
> 
> We should do the same for HyPixel

No.

**Comment by romangraef** [comment]
> No.

so sad

**Comment by walkerselby** [comment]
>  We should do the same for HyPixel
Screenshot 2022-10-04 at 1 09 19 PM
No.

**Comment by IRONM00N** [comment]
> >  Standardized spelling of SkyBlock to "SkyBlock", as indicated by Hypixel
> 
> 
> 
> We should do the same for HyPixel
> 
> 

:trollface:

File: src/main/java/io/github/moulberry/notenoughupdates/commands/misc/CosmeticsCommand.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public CosmeticsCommand() {
 	public void processCommand(ICommandSender sender, String[] args) throws CommandException {
 		if (!OpenGlHelper.isFramebufferEnabled() && NotEnoughUpdates.INSTANCE.config.notifications.doFastRenderNotif) {
 			Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(EnumChatFormatting.RED +
-				"NEU cosmetics do not work with OF Fast Render. Go to ESC > Options > Video Settings > Performance > Fast Render to disable it."));
+				"NEU Cosmetics do not work with Optifine Fast Render. Go to ESC > Options > Video Settings > Performance > Fast Render to disable it."));
```
Comment (MicrocontrollersDev) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/misc/CosmeticsCommand.java:42]:
```suggestion
				"NEU Cosmetics do not work with OptiFine Fast Render. Go to ESC > Options > Video Settings > Performance > Fast Render to disable it."));
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/profile/ViewProfileCommand.java [hunk 2]
```diff
@@ -39,18 +39,18 @@ public class ViewProfileCommand extends ClientCommandBase {
 	public static final Consumer<String[]> RUNNABLE = (args) -> {
 		if (!OpenGlHelper.isFramebufferEnabled()) {
 			Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(EnumChatFormatting.RED +
-				"Some parts of the profile viewer do not work with OF Fast Render. Go to ESC > Options > Video Settings > Performance > Fast Render to disable it."));
+				"Some parts of the profile viewer do not work with Optifine Fast Render. Go to ESC > Options > Video Settings > Performance > Fast Render to disable it."));
```
Comment (MicrocontrollersDev) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/profile/ViewProfileCommand.java:42]:
```suggestion
				"Some parts of the profile viewer do not work with OptiFine Fast Render. Go to ESC > Options > Video Settings > Performance > Fast Render to disable it."));
```

**Review by MicrocontrollersDev — CHANGES_REQUESTED** [review]
also i agree with the jani i don't care what hypixel says, "SkyBlock" is wrong.

**Comment by walkerselby** [comment]
> also i agree with the jani i don't care what hypixel says, "SkyBlock" is wrong.

I am fine with either way, we just had 33% skyblock, 33% Skyblock and 33# SkyBlock, so I was going for something consistent. If we favor Skyblock,
I can change it to that when I am back at my desk.

**Review by MicrocontrollersDev — DISMISSED** [review]
while i disagree with hypixel’s direction with SkyBlock, everything looks good

**Review by hannibal002 — APPROVED** [review]

**Review by hannibal002 — CHANGES_REQUESTED** [review]
merge needed

**Review by Lulonaut — APPROVED** [review]
HyPixel SkyBlock

**Review by hannibal002 — APPROVED** [review]
HyBlock SkyPixel

**Review by MicrocontrollersDev — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Pull request merged**